tests/robot-tests/tests/libs/common.robot

```diff
@@ -93,7 +93,7 @@ user opens chrome headless
     Call Method    ${c_opts}    add_argument    disable-logging
 
     Create Webdriver    Chrome    crm_alias    chrome_options=${c_opts}
-    setup chromedriver
+    enable basic auth headers
```

Comment:
Now that we're enabling/disabling the headers on a case-by-case basis, would suggest removing these as the defaults across the various `user opens chrome...` keywords